huxley/core/models.py

```diff
@@ -394,6 +394,11 @@ class Delegate(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     summary = models.TextField(default='', blank=True, null=True)
 
+    friday_attendance = models.BooleanField(default=False)
```

Comment:
Is attendance per-delegate, or per assignment? Like, when the chair calls out a country and only one delegate responds, how do you know who it is?